source/comply/audit-log.rst

```diff
@@ -1,369 +1,293 @@
-       "max_age", "number", "0", "Maximum number of days to retain old log files based on the timestamp encoded in their filename. 0 means do not remove old log files based on age."
-       "max_backups", "number", "0", "Maximum number of old log files to retain.  0 means retain all old log files (though max_age may still cause them to get deleted.)"
-       "compress", "bool", "false", "When true, the rotated log files will be compressed using gzip."
-
-    
-**TCP configuration options:** 
-
-.. csv-table::
-    :header: "Key", "Type", "Default", "Description"
-       
-       "host", "string", "  ", "IP or domain name of server to receive log records."
-       "port", "number", " ", "Port number for server receiving log records."
-       "tls", "bool", "false", "When true, a TLS connection will be created."
-       "cert", "string", "  ", "Path to a cert file (.pem) to be used when establishing a TLS connection."
-       "insecure", "bool", "false", "When true, any certificate will be accepted and any host name in that certificate. Should be used only in testing environments."
-
-    
-To temporarily disable a log target its type can be set to "none".
-
-Log formatters
-~~~~~~~~~~~~~~~
-
-Mattermost currently supports three log formats: plain, JSON, and `GELF <https://docs.graylog.org/en/4.0/pages/gelf.html>`__.
-
-**Plain configuration options:**
-
-.. csv-table::
-    :header: "Key", "Type", "Default", "Description"
- 
-       "disable_timestamp", "bool", "alse", "Disables output of the timestamp."
-       "disable_level", "bool", "false", "Disables output of the level name."
-       "disable_msg", "bool", "false", "Disables output of the message text."
-       "disable_fields", "bool", "false", "Disables output of all fields."
-       "disable_stacktrace", "bool", "false", "Disables output of stack traces."
-       "delim", "string", "single space", "Delimiter placed between fields."
-       "min_level_len", "number", "0", "Minimum level name length. If the level name is less than the minimum it will be padded with spaces."
-       "min_msg_len", "number", "0", "Minimum msg length. If the msg text is less than the minimum it will be padded with spaces."
-       "timestamp_format", "string", "2006-01-02 15:04:05.000 Z07:00", "Format for timestamps. See `format <https://golang.org/pkg/time/#Time.Format>`_ for format details."
-       "line_end", "string", "\n ", "Alternative end of line character(s)."
-       "enable_color", "bool", "false", "Enables color for targets that support color output."
+The following audit log levels are available:
 
-**JSON configuration options:**
++--------+-----------------------+--------------------+
+| **ID** | **Name**              | **Description**    |
++--------+-----------------------+--------------------+
+| 100    | ``audit-api``         | API events         |
++--------+-----------------------+--------------------+
+| 101    | ``audit-content``     | Content changes    |
```

Comment:
Can we add a note regarding `audit-content` level denoting that it can generate a *lot* more records than the others?